src/app/personside/infotabs/meldinger/traadvisning/Enkeltmelding.tsx

```diff
@@ -89,20 +95,26 @@ function EnkeltMelding(props: Props) {
     const fraNav = erMeldingFraNav(props.melding.meldingstype);
     const topptekst = meldingstittel(props.melding);
     const datoTekst = formatterDatoTid(props.melding.opprettetDato);
-    const skrevetAv = saksbehandlerTekst(props.melding.skrevetAv);
     const highlightRule = createDynamicHighligtingRule(props.sokeord.split(' '));
 
     return (
         <div className="snakkeboble_ikoner">
             <Snakkeboble pilHoyre={fraNav} ikonClass={fraNav ? 'nav-ikon' : 'bruker-ikon'}>
                 <SnakkebobleWrapper>
                     <Topptekst>
-                        <Inline>
-                            <Element>{topptekst}</Element>
+                        <SpaceBetween>
+                            <BoldTekstomrade rules={[ParagraphRule, highlightRule, LinkRule]}>
+                                {topptekst}
+                            </BoldTekstomrade>
                             <MeldingLestEtikett melding={props.melding} />
-                        </Inline>
-                        <Normaltekst>{datoTekst}</Normaltekst>
-                        <Normaltekst>Skrevet av: {skrevetAv}</Normaltekst>
+                        </SpaceBetween>
+                        <Tekstomrade>{datoTekst}</Tekstomrade>
+                        <SkrevetAvStyle>
+                            <span>Skrevet av:</span>
```

Comment:
Kan vi heller lage en komponent for denne? Så slipper vi å bruke `> *:first-child`-selectoren også. Ikke så fan av refaktoreringer.